src/GovUk.Education.ExploreEducationStatistics.Content.Model/GlossaryEntries.cs

```diff
@@ -0,0 +1,23 @@
+#nullable enable
+
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+{
+    public class GlossaryEntries
+    {
+        public Guid Id { get; set; }
+
+        public string Title { get; set; }
+
+        public string Slug { get; set; }
```

Comment:
What is slug going to be used for? Html anchor?

Reply:
It is used for the id tag on the glossary page. We need to keep this for links to specific glossary entries.